_source/logzio_collections/_log-sources/dotnet.md

```diff
@@ -320,46 +320,64 @@ namespace dotnet_log4net
 ```
 
 ### Serverless platforms
-If you’re using a serverless function, you’ll need to call the appender's flush methods at the end of the function run to make sure the logs are sent before the function finishes its execution.
+If you’re using a serverless function, you’ll need to call the appender's flush methods at the end of the function run to make sure the logs are sent before the function finishes its execution. You’ll also need to create a static appender in the Startup.cs file so each invocation will use the same appender. 
```

Comment:
Since the script is relevant specifically for Azure function, please have it in a separate sub-section of Serverless platforms :)